portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -28,10 +28,13 @@
     // The options marked as 'liked' by the user.
     private ArrayList<String> likedOptions;
     private String text;
-    public Comment(String newAuthor, int newRate, ArrayList<String> newLikedOptions, String newText){
+    //Timestamp to enable sorting the comments
+    private final long timestamp;
+    public Comment(String newAuthor, int newRate, ArrayList<String> newLikedOptions, String newText, long newTimestamp) {
```

Comment:
I suggest reading about Builder design pattern in Java:

https://dzone.com/articles/design-patterns-the-builder-pattern

It's optional and might be out of scope for the portfolio project, but it's good to know for future reference :)